src/agent/multitype_queue/src/sqlitestorage.cpp

```diff
@@ -6,7 +6,38 @@
 #include <exception>
 #include <iostream>
 
-SQLiteStorage::SQLiteStorage(const std::string& dbName, const std::vector<std::string> tableNames)
+std::string SQLiteStorage::GenerateQuery(const std::string& formatString, const std::vector<std::string>& params)
```

Comment:
Isn't it possible to use `fmt` here?

Reply:
Can't we make SQL queries `constexpr`?
https://github.com/wazuh/wazuh-agent/pull/101/files#diff-80d1e3e3f3442f1af6f1658d4ccc5cd2a7815de0e850484ede01b034a778c43dR5

Reply:
Found this workaround, it may work https://godbolt.org/z/ebaKxv9Gh

Reply:
It works. Done.